higlass/tilesets.py

```diff
@@ -5,8 +5,10 @@
 
 import clodius.tiles.utils as hgut
```

Comment:
These imports are getting out of hand and are unreadable imho. Maybe they should be migrated into the functions that use them.